build/apt/apt.lockfile

```diff
@@ -0,0 +1,75 @@
+47cd7cb9bf37f657993a6ef2f24d4943ded5ef616d866f2d43bdf5e5b3abb6ea  ./apt/packages/binutils-common_2.35.2-2_amd64.deb
```

Comment:
Is there a way for us to not commit the `deb` files in our repository?

Reply:
If `git-lfs` is used, we can clone the repo without git-lfs files using a command like:
`git clone --filter=blob:none <repository-url>`

Alternatively they can live in a bucket, or a separate repo and get pulled in using `gitsubmodules`